main.go

```diff
@@ -304,6 +304,10 @@ func startOutputs() error {
 		outputHandler = &S3Output{}
 	case SyslogOutputType:
 		outputHandler = &SyslogOutput{}
+	case HttpOutputType:
+		var f HttpFormatter
+		f = &SensorAlertsFormatter{ }
+		outputHandler = &HttpOutput{ formatter : f }
```

Comment:
Same comment about templating here- we would definitely need some sort of template system before accepting the patch as-is.